src/utils/config_parser/index.ts

```diff
@@ -36,25 +37,13 @@ export const parseOwnershipRules = (
   return ownershipRules;
 };
 
-const getDefaultOwnershipRules = (): OwnershipRules => {
-  const ownershipRules: OwnershipRules = {
-    directoryMatchingRules: [
-      {
-        name: "Default matching rule",
-        path: "playground/{{username}}/**/*",
-      },
-    ],
-  };
-  return ownershipRules;
-};
-
 export const getOwnershipRules = async (
-  context: Context,
+  context: Context | null,
 ): Promise<OwnershipRules> => {
-  const config = (await context.config("approveman.yml")) as unknown[];
-  context.log.info(`Found config: ${JSON.stringify(config)}`);
-  if (config !== null) {
-    return parseOwnershipRules(config, context);
+  const config = await context?.config("approveman.yml");
+  context?.log.info(`Found config: ${JSON.stringify(config)}`);
+  if (config != null) {
```

Comment:
Codacy found an issue: [Expected '!==' and instead saw '!='.](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6092258)